lib/middleman-cdn/cdns/cloudflare.rb

```diff
@@ -15,7 +17,8 @@ def self.example_configuration_elements
           client_api_key: ['"..."', "# default ENV['CLOUDFLARE_CLIENT_API_KEY']"],
           email: ['"..."', "# default ENV['CLOUDFLARE_EMAIL']"],
           zone: ['"..."', ""],
-          base_urls: [['http://example.com', 'https://example.com'], ""]
+          base_urls: [['http://example.com', 'https://example.com'], ""],
+          invalidate_zone_for_many_files: [true, "# default true"]
```

Comment:
This says `true` by default, but I found it wasn't working until I explicitly added it to my config. Should I hope an issue?